### DISCUSSION TITLE
how to run examples/water case at stage of dp test?

### DISCUSSION THREAD

**Discussion by tonystarkiss** [deepmodeling/deepmd-kit#694] [Q&A]
I have trained the water model with the examples fold case, and got a graph.pb, follow the guide: https://deepmd.readthedocs.io/en/master/use-deepmd-kit.html, I should perform the test stage:
dp test -m graph.pb -s /path/to/system -n 30
my question is: How to set paramter:  _**-s /path/to/system**_, in this case?

**Comment by njzjz** [comment]
It can be the same as the systems you set in your training parameters, such as `../data/`
https://github.com/deepmodeling/deepmd-kit/blob/773ed37ce4f95d04ce13ca3a1348fe4295f9d7d8/examples/water/train/water_se_a.json#L45

**Reply by tonystarkiss** [reply]
thanks
